README.md

```diff
@@ -3,21 +3,21 @@ cob_control
 
 ## ROS Distro Support
 
-|         | Indigo | Jade | Kinetic |
-|:-------:|:------:|:----:|:-------:|
-| Branch  | [`indigo_dev`](https://github.com/ipa320/cob_control/tree/indigo_dev) | [`indigo_dev`](https://github.com/ipa320/cob_control/tree/indigo_dev) | [`kinetic_dev`](https://github.com/ipa320/cob_control/tree/kinetic_dev) |
-| Status  |  supported | not supported |  supported |
-| Version | [version](http://repositories.ros.org/status_page/ros_indigo_default.html?q=cob_control) | [version](http://repositories.ros.org/status_page/ros_jade_default.html?q=cob_control) | [version](http://repositories.ros.org/status_page/ros_kinetic_default.html?q=cob_control) |
+|         | Indigo | Jade | Kinetic | Melodic |
```

Comment:
`Lunar` column missing?

Reply:
Lunar is EOL. But so are Indigo and Jade.

Reply:
it's just for the sake of completeness...